mico-core/src/main/java/io/github/ust/mico/core/web/ApplicationController.java

```diff
@@ -256,6 +239,96 @@
         return ResponseEntity.ok(new Resource<>(applicationDeploymentInformation));
     }
 
+    @GetMapping("/{" + PATH_VARIABLE_SHORT_NAME + "}")
+    public ResponseEntity<Resources<Resource<MicoApplication>>> getApplicationsByShortName(@PathVariable(PATH_VARIABLE_SHORT_NAME) String shortName) {
+        List<MicoApplication> micoApplicationList = applicationRepository.findByShortName(shortName);
+
+        List<Resource<MicoApplication>> applicationResourceList = getApplicationResourceList(micoApplicationList);
+
+        return ResponseEntity.ok(
+            new Resources<>(applicationResourceList,
+                linkTo(methodOn(ApplicationController.class).getApplicationsByShortName(shortName)).withSelfRel()));
+    }
+
+    @PatchMapping("/{" + PATH_VARIABLE_SHORT_NAME + "}/{" + PATH_VARIABLE_VERSION + "}/" + PATH_SERVICES)
```

Comment:
This should be a PostMapping.
The PATCH method would require a providing a set of changes (such as adding or removing a service) instead of just a service by specification [RFC 5789](https://tools.ietf.org/html/rfc5789#page-2).
POST also matches a generic "add this" semantic (see [specification](https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html) and [prior discussion](https://github.com/UST-MICO/mico/pull/270#discussion_r251195349)) without the need to provide a spechial patch document.